Backpack/src/main/java/net/skyscanner/backpack/bottomsheet/BpkBottomSheetBehaviour.kt

```diff
@@ -56,7 +56,7 @@ class BpkBottomSheetBehaviour<V : View>(context: Context, attrs: AttributeSet? =
       override fun onSlide(bottomSheet: View, slideOffset: Float) {
         updateBackground(slideOffset)
       }
-    })
+    },)
```

Comment:
there's some of these that feel wrong - can we also enforce putting the ) onto the next line?

Reply:
I found no way to do that so far